src/components/stepforms/schemas/rollup.ts

```diff
@@ -0,0 +1,95 @@
+import { addNotInColumnNamesConstraint, StepFormType } from './utils';
+
+const schema = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Rollup step',
+  type: 'object',
+  properties: {
+    name: {
+      type: 'string',
+      enum: ['rollup'],
+    },
+    hierarchy: {
+      type: 'array',
+      items: {
+        type: 'string',
+        minLength: 1,
+      },
+      title: 'hierarchal columns',
+      description: 'The hierarchy of columns, from lowest to highest level',
+      attrs: {
+        placeholder: 'Add columns',
+      },
+    },
+    aggregations: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          column: {
+            type: 'string',
+            minLength: 1,
```

Comment:
Would it be possible to have no aggregations columns?
I imagine people wanted to create a referential, and therefore having only labels and no value columns.
It shouldn't be a problem in the query I think, and it would be a shame to have to create a dummy column with only 0 for this to work.